lib/rules/require-unicode-regexp.js

```diff
@@ -112,11 +178,11 @@ module.exports = {
                                             return fixer.insertTextAfter(
                                                 penultimateToken,
                                                 astUtils.isCommaToken(penultimateToken)
-                                                    ? ' "u",'
-                                                    : ', "u"'
+                                                    ? ` "'${replaceFlag}'",`
```

Comment:
```suggestion
                                                    ? ` "${replaceFlag}",`
```

These would be extra quotes inside the produced string literal. Can you also add a test for this case? (apparently, this code path wasn't tested at all).

Reply:
Added, thanks.